source/PluginDev/Assets/GooglePlayGames/Platforms/Native/NativeClient.cs

```diff
@@ -341,8 +341,8 @@ public string GetServerAuthCode()
         public void GetAnotherServerAuthCode(bool reAuthenticateIfNeeded,
                                              Action<string> callback)
         {
-            mTokenClient.GetAnotherServerAuthCode(reAuthenticateIfNeeded,
-                                                  callback);
+            PlayGamesHelperObject.RunOnGameThread(() => mTokenClient.GetAnotherServerAuthCode(reAuthenticateIfNeeded,
```

Comment:
It seems we have similar code in AndroidTokenClient.cs for FetchTokens
```
        public void FetchTokens(bool silent, Action<int> callback)
        {
            PlayGamesHelperObject.RunOnGameThread(() => DoFetchToken(silent, callback));
        }

        internal void DoFetchToken(bool silent, Action<int> callback)
        { ... }
```
Probably, we should have there the same logic for 'GetAnotherServerAuthCode'. I mean adding
`internal void DoGetAnotherServerAuthCode(bool silent, Action<int> callback)`
and calling it using `PlayGamesHelperObject.RunOnGameThread`